src/components/simple-color-picker/simple-color-picker.test.tsx

```diff
@@ -0,0 +1,496 @@
+  const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
+  const onChange = jest.fn();
+  render(
+    <SimpleColorPicker
+      onChange={onChange}
+      maxWidth="100"
+      childWidth="100"
+      legend="SimpleColorPicker Legend"
+      name="test"
+    >
+      <SimpleColor value="#00A376" />
+      <SimpleColor value="#0073C1" />
+      <SimpleColor value="#582C83" />
+    </SimpleColorPicker>
+  );
+
+  screen.getAllByRole("radio")[1].focus();
+  await user.keyboard("{ArrowDown}");
+
+  expect(onChange).toHaveBeenCalledTimes(1);
+  expect(onChange).toHaveBeenCalledWith(
+    expect.objectContaining({
+      target: expect.objectContaining({ value: "#582C83" }),
+    })
+  );
+  expect(screen.getAllByRole("radio")[2]).toBeFocused();
+});
+
+test("when focus is already on the bottom row, pressing the down arrow key does not change selection", async () => {
+  const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
+  const onChange = jest.fn();
+  render(
+    <SimpleColorPicker
+      onChange={onChange}
+      maxWidth="100"
+      childWidth="40"
+      legend="SimpleColorPicker Legend"
+      name="test"
+    >
+      <SimpleColor value="#00A376" />
+      <SimpleColor value="#0073C1" />
+      <SimpleColor value="#582C83" />
+      <SimpleColor value="#ABCDEF" />
+    </SimpleColorPicker>
+  );
+
+  screen.getAllByRole("radio")[3].focus();
+  await user.keyboard("{ArrowDown}");
+
+  expect(onChange).not.toHaveBeenCalled();
+  expect(screen.getAllByRole("radio")[3]).toBeFocused();
+});
+
+test("focus is not changed if a non-arrow key is pressed", async () => {
+  const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
+  const onChange = jest.fn();
+  render(
+    <SimpleColorPicker
+      onChange={onChange}
+      legend="SimpleColorPicker Legend"
+      name="test"
+    >
+      <SimpleColor value="#00A376" />
+      <SimpleColor value="#0073C1" />
+      <SimpleColor value="#582C83" />
+    </SimpleColorPicker>
+  );
+
+  screen.getAllByRole("radio")[0].focus();
+  await user.keyboard("{Control}");
+
+  expect(onChange).not.toHaveBeenCalled();
+  expect(screen.getAllByRole("radio")[0]).toBeFocused();
+});
+
+test("the `onBlur` callback prop should not be called if focus moves from one color input to another", async () => {
+  const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
+  const onBlur = jest.fn();
+  render(
+    <SimpleColorPicker
+      onBlur={onBlur}
+      legend="SimpleColorPicker Legend"
+      name="test"
+    >
+      <SimpleColor value="#00A376" />
+      <SimpleColor value="#0073C1" />
+      <SimpleColor value="#582C83" />
+    </SimpleColorPicker>
+  );
+
+  await user.click(screen.getAllByRole("radio")[0]);
+  await user.click(screen.getAllByRole("radio")[1]);
+  jest.runAllTimers();
+
+  expect(onBlur).not.toHaveBeenCalled();
+});
+
+// mostly kept for coverage - it's not clear what value this particular test really provides
```

Comment:
Looking at this test, I _think_ it seems likely it'll be coverage for the `isBlurBlocked` behaviour that we'll be deprecating soon-ish hopefully. It might be an idea for whoever does that work to remove this test to see if coverage is still an issue or not.